extensions/parquet/base/src/main/java/io/deephaven/parquet/base/InvalidParquetFileException.java

```diff
@@ -0,0 +1,7 @@
+package io.deephaven.parquet.base;
+
+public class InvalidParquetFileException extends ParquetFileReaderException {
```

Comment:
Javadoc? alternatively, change the test to catch any ParquetFileReaderException and just check the `message` to see if it is this kind of failure?

Reply:
I feel that a different exception is less error prone than relying on the message string. So added a Javadoc for now. Hope that's okay.